src/adams_methods.jl

```diff
@@ -0,0 +1,637 @@
+"""
+    [T,X] = ode_ms(ODEFUN, Y0, TSPAN,ORDER=4) with TSPAN = [T0:dt:TFINAL]
```

Comment:
This is a bit matlab-y, at least change it to:
`t,y = ode_ms(ODEFUN, Y0, TSPAN, ORDER=4)`
but I would use lower-case.